src/app.rs

```diff
@@ -96,6 +96,20 @@ impl TaskStore {
             task.tags.retain(|f| f != &tag_id);
         }
     }
+
+    pub fn sort(&mut self) {
+        self.tasks.sort_unstable_by_key(|t| t.priority);
+        self.tasks.reverse();
+    }
+
+    pub fn add_task(&mut self, task: Task) {
```

Comment:
Is this used anywhere? It should probably be used within `restore_task` and the `a` action in `main_screen`